2020-04-12T23:17:25.3601202+07:00 0HLUUVJU3OIO7:00000001 [INF] Request starting HTTP/1.1 DEBUG http://localhost:52653/  0 (ca22a1cb)
2020-04-12T23:17:25.3601201+07:00 0HLUUVJU3OIO6:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:44364/swagger/index.html   (ca22a1cb)
2020-04-12T23:17:26.6328560+07:00 0HLUUVJU3OIO7:00000001 [INF] Request finished in 1256.6758ms 200  (791a596a)
2020-04-12T23:17:28.4086664+07:00 0HLUUVJU3OIO6:00000001 [INF] Request finished in 3080.3088ms 200 text/html (791a596a)
2020-04-12T23:17:34.6079854+07:00 0HLUUVJU3OIO7:00000002 [INF] Request starting HTTP/1.1 GET http://localhost:44364/swagger/v1/swagger.json   (ca22a1cb)
2020-04-12T23:17:44.2814684+07:00 0HLUUVJU3OIO7:00000002 [INF] Request finished in 9679.9169ms 200 application/json;charset=utf-8 (791a596a)
2020-04-12T23:35:58.4237783+07:00 0HLUUVJU3OIO8:00000001 [INF] Request starting HTTP/1.1 POST http://localhost:44364/api/vi-VN/account/register application/json-patch+json 114 (ca22a1cb)
2020-04-12T23:35:59.0583362+07:00 0HLUUVJU3OIO8:00000001 [INF] Route matched with "{action = \"Register\", controller = \"Account\"}". Executing action "ChicilonMedia.API.Controllers.AccountController.Register (ChicilonMedia.API)" (a44c0341)
2020-04-12T23:35:59.6641511+07:00 0HLUUVJU3OIO8:00000001 [INF] Executing action method "ChicilonMedia.API.Controllers.AccountController.Register (ChicilonMedia.API)" with arguments (["ChicilonMedia.Data.ViewModels.RegisterViewModel"]) - Validation state: Valid (4e3479ed)
2020-04-12T23:36:02.7583017+07:00 0HLUUVJU3OIO8:00000001 [INF] Executed action "ChicilonMedia.API.Controllers.AccountController.Register (ChicilonMedia.API)" in 3693.1989ms (afa2e885)
2020-04-12T23:36:03.7626975+07:00 0HLUUVJU3OIO8:00000001 [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.Data.SqlClient.SqlException (0x80131904): Login failed for user ''.
   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling)
   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()
--- End of stack trace from previous location where exception was thrown ---
   at ChicilonMedia.API.Data.UserStore.FindByNameAsync(String normalizedUserName, CancellationToken cancellationToken) in C:\Users\latha\Desktop\src\ChicilonMedia\ChicilonMedia.API\Data\UserStore.cs:line 73
   at Microsoft.AspNetCore.Identity.UserManager`1.FindByNameAsync(String userName)
   at Microsoft.AspNetCore.Identity.UserValidator`1.ValidateUserName(UserManager`1 manager, TUser user, ICollection`1 errors)
   at Microsoft.AspNetCore.Identity.UserValidator`1.ValidateAsync(UserManager`1 manager, TUser user)
   at Microsoft.AspNetCore.Identity.UserManager`1.ValidateUserAsync(TUser user)
   at Microsoft.AspNetCore.Identity.UserManager`1.CreateAsync(TUser user)
   at Microsoft.AspNetCore.Identity.UserManager`1.CreateAsync(TUser user, String password)
   at ChicilonMedia.API.Controllers.AccountController.Register(RegisterViewModel model) in C:\Users\latha\Desktop\src\ChicilonMedia\ChicilonMedia.API\Controllers\AccountController.cs:line 93
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.MiddlewareFilterBuilder.<>c.<<BuildPipeline>b__8_0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Localization.RequestLocalizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.Invoke(HttpContext context)
ClientConnectionId:56ee01f6-c047-4701-b2b0-2c5d95e77303
Error Number:18456,State:1,Class:14
2020-04-12T23:36:03.8523837+07:00 0HLUUVJU3OIO8:00000001 [INF] Request finished in 5457.3124ms 500 application/json (791a596a)
2020-04-12T23:37:08.4319524+07:00 0HLUUVJU3OIO9:00000001 [INF] Request starting HTTP/1.1 POST http://localhost:44364/api/en-US/account/register application/json-patch+json 114 (ca22a1cb)
2020-04-12T23:37:08.4556189+07:00 0HLUUVJU3OIO9:00000001 [INF] Route matched with "{action = \"Register\", controller = \"Account\"}". Executing action "ChicilonMedia.API.Controllers.AccountController.Register (ChicilonMedia.API)" (a44c0341)
2020-04-12T23:37:08.5332064+07:00 0HLUUVJU3OIO9:00000001 [INF] Executing action method "ChicilonMedia.API.Controllers.AccountController.Register (ChicilonMedia.API)" with arguments (["ChicilonMedia.Data.ViewModels.RegisterViewModel"]) - Validation state: Valid (4e3479ed)
2020-04-12T23:37:08.6859697+07:00 0HLUUVJU3OIO9:00000001 [INF] Executed action "ChicilonMedia.API.Controllers.AccountController.Register (ChicilonMedia.API)" in 228.2139ms (afa2e885)
2020-04-12T23:37:08.7116160+07:00 0HLUUVJU3OIO9:00000001 [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.Data.SqlClient.SqlException (0x80131904): Login failed for user ''.
   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling)
   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()
--- End of stack trace from previous location where exception was thrown ---
   at ChicilonMedia.API.Data.UserStore.FindByNameAsync(String normalizedUserName, CancellationToken cancellationToken) in C:\Users\latha\Desktop\src\ChicilonMedia\ChicilonMedia.API\Data\UserStore.cs:line 73
   at Microsoft.AspNetCore.Identity.UserManager`1.FindByNameAsync(String userName)
   at Microsoft.AspNetCore.Identity.UserValidator`1.ValidateUserName(UserManager`1 manager, TUser user, ICollection`1 errors)
   at Microsoft.AspNetCore.Identity.UserValidator`1.ValidateAsync(UserManager`1 manager, TUser user)
   at Microsoft.AspNetCore.Identity.UserManager`1.ValidateUserAsync(TUser user)
   at Microsoft.AspNetCore.Identity.UserManager`1.CreateAsync(TUser user)
   at Microsoft.AspNetCore.Identity.UserManager`1.CreateAsync(TUser user, String password)
   at ChicilonMedia.API.Controllers.AccountController.Register(RegisterViewModel model) in C:\Users\latha\Desktop\src\ChicilonMedia\ChicilonMedia.API\Controllers\AccountController.cs:line 93
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.MiddlewareFilterBuilder.<>c.<<BuildPipeline>b__8_0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Localization.RequestLocalizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.Invoke(HttpContext context)
ClientConnectionId:6baff28e-702f-40b5-8063-2f4371fd954a
Error Number:18456,State:1,Class:14
2020-04-12T23:37:08.7162597+07:00 0HLUUVJU3OIO9:00000001 [INF] Request finished in 303.8809ms 500 application/json (791a596a)
2020-04-12T23:38:50.4246957+07:00 0HLUUVJU3OIOA:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:44364/api/function   (ca22a1cb)
2020-04-12T23:38:50.4357229+07:00 0HLUUVJU3OIOA:00000001 [INF] Route matched with "{action = \"Get\", controller = \"Function\"}". Executing action "ChicilonMedia.API.Controllers.FunctionController.Get (ChicilonMedia.API)" (a44c0341)
2020-04-12T23:38:50.4449260+07:00 0HLUUVJU3OIOA:00000001 [INF] Executing action method "ChicilonMedia.API.Controllers.FunctionController.Get (ChicilonMedia.API)" - Validation state: Valid (dad538d7)
2020-04-12T23:38:50.5620988+07:00 0HLUUVJU3OIOA:00000001 [INF] Executed action "ChicilonMedia.API.Controllers.FunctionController.Get (ChicilonMedia.API)" in 124.451ms (afa2e885)
2020-04-12T23:38:50.5722660+07:00 0HLUUVJU3OIOA:00000001 [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.Data.SqlClient.SqlException (0x80131904): Login failed for user ''.
   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling)
   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()
--- End of stack trace from previous location where exception was thrown ---
   at ChicilonMedia.API.Controllers.FunctionController.Get() in C:\Users\latha\Desktop\src\ChicilonMedia\ChicilonMedia.API\Controllers\FunctionController.cs:line 35
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.Invoke(HttpContext context)
ClientConnectionId:9a85e0d0-03fc-43c6-9502-1473cc5d17ba
Error Number:18456,State:1,Class:14
2020-04-12T23:38:50.5769355+07:00 0HLUUVJU3OIOA:00000001 [INF] Request finished in 162.0595ms 500 application/json (791a596a)
